src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -1156,7 +1174,9 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 			.map((action: ActionItem) => {
 				const existsInSortMap = !isNil(sortMap.find((d) => d.id === action.id));
 				const isPrimary = sortMap.find((d) => action.id === d.id)?.isPrimary ?? false;
-				const showExport = action.id === MediaActions.Export && canDownloadMetadata;
+				const showExport =
+					(action.id === MediaActions.Export && canDownloadMetadata) ||
+					(action.id === MediaActions.Export && canDownloadNewspaper);
```

Comment:
This is the same code but more consise:
```suggestion
				const showExport =
					action.id === MediaActions.Export && (canDownloadMetadata || canDownloadNewspaper);
```